Makefile

```diff
@@ -541,18 +541,21 @@ lint-spell:
 
 lint: lint-js lint-sass lint-cpp lint-html lint-spell
 
-ELECTRON_MOCHA_OPTIONS=--recursive --reporter spec
+MOCHA_OPTIONS=--recursive --reporter spec
+
+test-spectron:
```

Comment:
This should also be added to the `TARGETS` list on line 370

Reply:
Added!